docs/source/dependencies/seqrepo.rst

```diff
@@ -0,0 +1,190 @@
+.. _seqrepo-setup:
+
+AnyVar SeqRepo Setup
+!!!!!!!!!!!!!!!!!!!!
+
+SeqRepo is a repository system designed to store and manage biological sequence data efficiently.
+
+Recommended: Local Installation
+===============================
+
+The simplest and most efficient method for most users is a direct local file installation.
+
+
+1. Download the SeqRepo Archive
+-------------------------------
+
+::
+
+    curl -O https://storage.googleapis.com/clingen-public/seqrepo_2024-12-20.tar.gz
```

Comment:
@theferrit32 is it okay if we expose this to the public or should it really only be used for hackathons?

Reply:
I would prefer we remove this. If I make a copy of it in cloudflare we could point to that one instead